.github/workflows/test-prepare-runner.yml

```
@@ -53,3 +54,6 @@ jobs:
           if [ "${{ runner.os }}" != "Windows" ]; then
             ccache --version
           fi
+          if [[ "${{ runner.os }}" = "macOS" || "${{ runner.os }}" = "Windows" ]]; then
```

```suggestion
          if [ "${{ runner.os }}" != "Linux" ]; then
```

Keeping it same as previous statement